docs/src/main/paradox/release-notes/releases-1.0.md

```diff
@@ -0,0 +1,78 @@
+# Release Notes (1.0.x)
+
+## 1.0.3-M1
+This is milestone release and is aimed at testing some new support for users who want to add Pekko nodes to an existing Akka cluster. This support is experimental. This release should not be used in production.
+
+### Bug Fixes
+
+* Fix StackOverflowError in ByteStringBuilder.addAll ([PR903](https://github.com/apache/pekko/pull/903))
+* Fix issue with possible int overflow in snapshot interval calculation ([PR1088](https://github.com/apache/pekko/pull/1088))
```

Comment:
Maybe mention #1183
This is quite an annoying bug when running larger Pekko clusters in k8s.